_platforms/en/spid.md

```diff
@@ -197,6 +209,7 @@ SPID can be integrated in the websites of the Public Administration, but also on
 3. Use a test identity provider such as[spid-saml-check](https://github.com/italia/spid-saml-check){:target="_blank"} or [spid-testenv2](https://github.com/italia/spid-testenv2){:target="_blank"} to simulate the authentication flow and verify that your implementation is correct.
 4. Get in touch with other developers at [Slack](https://slack.developers.italia.it/){:target="_blank"}.
 5. Follow the accreditation procedure [described here](https://www.spid.gov.it/come-diventare-fornitore-di-servizi-pubblici-e-privati-con-spid){:target="_blank"}.
+6. If you have any further questions or are having problems with onboarding, please consult the [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add) page.
```

Comment:
```suggestion
6. If you have any further questions or are having problems with the procedure, please consult the [SPID HelpDesk](https://helpdesk.spid.gov.it/index.php?a=add) page.
```

We call it "accreditation procedure" in the previous point

Reply:
Looking at the helpdesk I'm wondering, is that the correct site for someone having troubles when onboarding?

![image](https://user-images.githubusercontent.com/788293/116993865-50983d80-acd8-11eb-8b44-c67185adbc61.png)

I wouldn't know what button to pick.

Reply:
Good question ... Anyway that's AgID's help desk page, I'd go further with this mention then come back on this topic in a different way

Reply:
> We call it "accreditation procedure" in the previous point

I made a commit that normalize this to "onboarding", does it sound good to you?